plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -80,6 +81,13 @@ public void testShowCreate()
         assertUpdate("DROP MATERIALIZED VIEW materialized_view_with_property");
     }
 
+    @Test
+    public void testSystemMaterializedViewProperties()
```

Comment:
no unit tests in the engine itself?

Reply:
Yes, I didn't find unit tests for other system.metadata tables either.
There are product tests covering it though.